scania-digital-design-system/tegel
# fix(header-dropdown): close on slotted items click

**Describe pull-request**  
Close the header dropdown after clicking on one of its children

**Solving issue**  
Fixes: [CDEP-2998](https://tegel.atlassian.net/browse/CDEP-2998)

**How to test**  
1. Go to the [this PR](https://github.com/scania-digital-design-system/tegel-react-demo/pull/178). It is built using develop. Observe what happens on clicking on one of the items in Header Dropdown, Dropdown menu stays opened. 
2. Go to [PR with this fix](https://github.com/scania-digital-design-system/tegel-react-demo/pull/179). Click the Header Dropdown and observe what is happening. Navigation should work and menu should close after clicking on one of its items.
3. Check the code in this PR if it makes sense.

**Checklist before submission**

**Suggested test steps**

**Additional context**  
Hard to test in Storybook, therefor I made packed version of code and made PRs for comparison in React Demo page. 
If this works, we can add the same behaviour to the Popover Canvas and Popover Menu, maybe even Header Launcher.


[CDEP-2998]: https://tegel.atlassian.net/browse/CDEP-2998?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
